### PR TITLE
[feature] make command `make` work, and add github workflows

### PR DESCRIPTION
**Pull request by yandongxiao**
see https://github.com/yandongxiao/starrocks-kubernetes-operator/actions for more debug information about github workflows

### CODE REVIEW

File: doc/cn_design.md
Comment (kevincai) [inline doc/cn_design.md:1]: may need to keep this doc for now, you can move it to a separate directory named `deprecated`.

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```
Comment (kevincai) [inline config/manager/manager.yaml:1]: what is this file for?

File: .github/workflows/ci-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: CI PIPELINE
+
+on:
+  push:
+    branches: ["feature/add-unit-test-workflow"]
```
[the remark this answers is not in the document]
Comment (yandongxiao) [reply .github/workflows/ci-pipeline.yml:5]: I Forgot to fix the  Github Event Part

File: config/manager/manager.yaml (hunk 1 above)
Comment (yandongxiao) [reply config/manager/manager.yaml:1]: just make the `kubectl kustomize .` command can work in  `starrocks-kubernetes-operator/config/manager`.

Comment (kevincai) [reply config/manager/manager.yaml:1]: ok

**Review by kevincai — APPROVED** [review]

**Pull request merged**